config.default.yaml

```diff
@@ -261,6 +261,7 @@ clustering:
   cluster_network:
     algorithm: kmeans
     feature: solar+onwind-time
+  exclude_carriers: [] # list of carriers which will not be aggregated. If empty, all carriers will be aggregated.
```

Comment:
Would it make sense to have separate `exclude_carrier` for `cluster_network` and `simplify_network`? I.e.

```yaml
clustering:
  simplify_network:
    exclude_carriers: []
  cluster_network:
    exclude_carriers: []
```
Gives you more control.